client/my-sites/upgrades/components/domain-warnings/index.jsx

```diff
@@ -179,15 +205,8 @@ export default React.createClass( {
 		}
 	},
 	render: function() {
-		let pipe = this.getPipe();
-
-		for ( let renderer of pipe ) {
-			let result = renderer();
-			if ( result ) {
-				return result;
-			}
-		}
-		return null;
+		const notices = this.getPipe().map( renderer => renderer() ).filter( notice => notice );
```

Comment:
What do you mean by a breaking change? That after this PR the warning will not work?